api/integration_test/form_version_test.go

```diff
@@ -0,0 +1,204 @@
+package integration_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http/httptest"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/18F/e-QIP-prototype/api"
+	"github.com/18F/e-QIP-prototype/api/http"
+	"github.com/18F/e-QIP-prototype/api/log"
+	"github.com/18F/e-QIP-prototype/api/mock"
+	"github.com/18F/e-QIP-prototype/api/postgresql"
+)
+
+type serviceSet struct {
+	env api.Settings
+	log api.LogService
+	// token    api.TokenService
+	db api.DatabaseService
+}
+
+func cleanTestServices() serviceSet {
+	env := &mock.Native{}
+	os.Setenv(api.LogLevel, "info")
+	env.Configure()
+
+	log := &log.Service{Log: log.NewLogger()}
+
+	db := &postgresql.Service{
+		Log: log,
+		Env: env,
+	}
+
+	db.Configure()
+
+	return serviceSet{
+		env,
+		log,
+		db,
+	}
+}
+
+func TestFormVersionReturned(t *testing.T) {
+
+	services := cleanTestServices()
+
+	// create/find test account (wow this should be in its own db)
+	account := api.Account{
+		Username:  "buzz1@example.com",
+		Email:     "buzz1@example.com",
+		SFType:    "SF86",
+		SFVersion: "2016-11",
+	}
+
+	_, err := account.Get(services.db, -1)
+	if err != nil {
+		if err.Error() == "pg: no rows in result set" {
```

Comment:
Can we pull this into a constant defined in the package somewhere?

Reply:
I'm making it an explicit error in my next PR, i'll port that over here.